cpp/include/cudf/scalar/scalar.hpp

```diff
@@ -230,7 +230,14 @@ class string_scalar : public scalar {
    * 
    * @param stream The CUDA stream to do the operation in
    */
-  std::string value(cudaStream_t stream = 0) const;
+  std::string to_string(cudaStream_t stream = 0) const;
+  
+  /**
+   * @brief Get the value of the scalar as a string_view
+   * 
+   * @param stream The CUDA stream to do the operation in
+   */
+  value_type value(cudaStream_t stream = 0) const { return value_type{data(), size()}; }
```

Comment:
I left the stream as a parameter because the value object is polymorphic.  However, we're not retrieving the actual data from the device to the host, so it doesn't actually need to use the parameter.